.github/workflows/build.yml

```diff
@@ -13,14 +13,16 @@ jobs:
       fail-fast: true
       matrix:
         os:
-          - macos-12
+          - macos-13
           - macos-11
         build_type:
           - Debug
           - Release
         qt:
-          - version: "5.15.2"
+          - version: "5.15.14"
```

Comment:
This release doe not exist as Open Source, it will just fallback to 15.2

Reply:
I saw that it was added to the opensource archive in May: https://download.qt.io/archive/qt/5.15/

Reply:
look at the build logs, aqtinstall does not know it

Reply:
also that link only provides sources, QtC did not release compiled versions of it

Reply:
It exists as open source bacause of their agreement but only as source, as binaries it's only available for comercial users.

Reply:
Well, that explains why such a seemingly recent build wouldn't work with MacOS 14...